install_config/install/advanced_install.adoc

```diff
@@ -1751,7 +1785,9 @@ state until its etcd instance is available.
 Some Ansible playbook bundles (APBs) also require a PV for their own usage in
 order to deploy. For example, each of the database APBs have two plans: the
 Development plan uses ephermal storage and does not require a PV, while the
-Production plan is persisted and does require a PV.
+Production plan is persisted and does require a PV. The PV depends on
+xref:../../install_config/persistent_storage/dynamically_provisioning_pvs.adoc#install-config-persistent-storage-dynamically-provisioning-pvs[dynamic
+storage provisioning] having been configured for the cluster.
```

Comment:
I would revert this change and go with the former text "Production plan is persisted and does require a PV."

As stated, some of the APBs do create a PVC and need it to be fulfilled.
There is _no_ requirement that the PV must be created from a dynamic provisioner, it is fine for an existing PV that was precreated to be fulfilled.  (We test with that scenario with oc cluster up which pre-creates a number of PVs)